tests/tools/test_dist_part.py

```diff
@@ -253,3 +120,39 @@ def rand_edges(num_src, num_dst, num_edges):
             orig_eids = load_tensors(fname)
             assert len(orig_eids.keys()) == 4
 
+            # Read edge_feat.dgl
+            fname = os.path.join(sub_dir, 'edge_feat.dgl')
+            assert os.path.isfile(fname)
+            tensor_dict = load_tensors(fname)
+            all_tensors = [
+                'paper:cites:paper/count',
+                'author:writes:paper/year',
+                'paper:rev_writes:author/year',
+            ]
+            assert tensor_dict.keys() == set(all_tensors)
+            for key in all_tensors:
+                assert isinstance(tensor_dict[key], torch.Tensor)
+
+            # Get orig_eids
+            orig_type_eids = g.edata['orig_id'].numpy()
+            orig_etype_ids = g.edata[dgl.ETYPE].numpy()
+
+            # Compare the data stored as edge features in this partition with the data
+            # from the original graph.
+            etype_names = list(edge_dict.keys())
+            for idx, etype_name in enumerate(etype_names):
```

Comment:
how about iterating over `tensor_dict` to avoid if check, `etype_name+'xxxx'`-like code?

Reply:
Done. as suggested.